be/functions/src/tests/onbarding-tests.sh

```
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# 스크립트 위치 기준으로 작업 디렉토리 이동
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)